Dockerfile

```diff
@@ -1,14 +1,9 @@
-FROM python:3.6.6
+FROM python:3.6.8-alpine
 
-RUN groupadd -r keeper && useradd --no-log-init -r -g keeper keeper
+RUN apk update&&apk --no-cache add --virtual build-dependencies build-base gcc wget git&&mkdir /keeper&&cd /keeper&&git clone https://github.com/makerdao/auction-keeper.git &&cd auction-keeper&&git submodule update --init --recursive&&pip install --no-cache-dir $(cat requirements.txt $(find lib -name requirements.txt | sort) | sort | uniq | sed 's/ *== */==/g') virtualenv&&apk del build-dependencies && apk add bash&&rm -rf /var/cache/apk/*
```

Comment:
git clone and submodule updates are automatically executed by travis, so when docker build is issued we are already in root of project with all submodules fetched. Hence we should not manually do this but just copy relevant dirs in Docker image. An improvement would be to just call newly added ./install.sh

I also advise on keeping the keeper user and running container as non root

Reply:
Maybe this only works for me, doesn't use travis, and builds from scratch. I will add the keeper account.